docs(android, player): add documentation for andriod player configura…

jspizziri opened the pull request:
…tion

mpivchev commented on docs/docs/api/constants/app-killed-playback-behavior.md:
```diff
@@ -0,0 +1,27 @@
+
+# App Killed Playback Behavior (android-only)
+
+All iOS Category types are made available through the named export `AppKilledPlaybackBehavior`:
```
Shouldn't this say Android Category?

jspizziri replied:
Dang... I shouldn't have merged so soon, sorry. I'll address all your feedback in a new PR.

mpivchev commented on docs/docs/api/objects/android-options.md:
```diff
@@ -0,0 +1,7 @@
+
+| Param | Type  | Default | Description |
+|-------|-------|---------|-------------|
+| `appKilledPlaybackBehavior` | [`AppKilledPlaybackBehavior`](../constants/app-killed-playback-behavior.md) | [`ContinuePlayback`](../constants/app-killed-playback-behavior#continueplayback-default.md) | Whether the audio playback notification is also removed when the playback stops. |
```
```suggestion
| `appKilledPlaybackBehavior` | [`AppKilledPlaybackBehavior`](../constants/app-killed-playback-behavior.md) | [`ContinuePlayback`](../constants/app-killed-playback-behavior#continueplayback-default.md) | Define how the audio playback should behave to removing the app from recents (killing it). |
```

mpivchev commented on docs/docs/basics/background-mode.md:
```diff
@@ -4,44 +4,63 @@ sidebar_position: 4
 
-React Native Track Player supports playing audio while your app is in the background on all supported platforms.
+React Native Track Player supports playing audio while your app is in the
+background on all supported platforms.
 
 ## Android
-Background audio playback works right out of the box. By default, the audio will continue to play, not only when the app is suspended in the background, but also after the app is closed by the user. If that is not the desired behavior, you can disable it with the `android.appKilledPlaybackBehavior` property in `updateOptions`. 
+Background audio playback works right out of the box. By default, the audio will
+ontinue to play, not only when the app is suspended in the background, but also
+after the app is closed by the user. If that is not the desired behavior, you
+can disable it with the `android.appKilledPlaybackBehavior` property in
+`updateOptions`.
```
Do we need these linebreaks?

mpivchev replied:
@dcvz what do you think?

jspizziri replied:
FWIW, I do think some character limit is a good thing (maybe 80 is too low). Otherwise you'll get code that will have lines that run on to infinity (logical evaluation expressions in conditionals/if statements is a good example).

mpivchev commented on docs/docs/basics/background-mode.md:
```diff
@@ -4,44 +4,63 @@ sidebar_position: 4
 ## Android
-Background audio playback works right out of the box. By default, the audio will continue to play, not only when the app is suspended in the background, but also after the app is closed by the user. If that is not the desired behavior, you can disable it with the `android.appKilledPlaybackBehavior` property in `updateOptions`. 
+Background audio playback works right out of the box. By default, the audio will
+ontinue to play, not only when the app is suspended in the background, but also
```
ontinue should be continue

mpivchev reviewed (approved):
LGTM! Some typos and suggestions commented

jspizziri commented:
@mpivchev addressed your comments here https://github.com/doublesymmetry/react-native-track-player/pull/1748/commits/e83e71cd0eb62aa0c7d2cf5dfb0df24149d9ce48
